Packs/CimTrak-SystemIntegrityAssurance/Playbooks/playbook-CimTrak_-_Example_-_Scan_Compliance_By_IP.yml

```diff
@@ -0,0 +1,165 @@
+contentitemexportablefields:
+  contentitemfields:
+    fromServerVersion: ""
+    itemVersion: ""
+    packID: 25043b93-4265-49a0-8858-04aed0ec4edd
```

Comment:
@kigerjoel I'm not sure why this is needed. Please remove the "packID" key.

Reply:
Fixed